sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -141,6 +141,10 @@ def _generate_sas_token(
     uri: str, policy: str, key: str, expiry: Optional[timedelta] = None
 ) -> AccessToken:
     """Create a shared access signiture token as a string literal.
+    :param str uri: The resource URI.
+    :param str policy: The name of the shared access policy.
+    :param str key: The shared access key.
+    :param timedelta or None expiry: The time period that the token is valid for.
```

Comment:
nit: is it supposed to be lower case?
```suggestion
    :param timedelta or none expiry: The time period that the token is valid for.
```